scalding-thrift-macros/src/main/scala/com/twitter/scalding/thrift/macros/RequiredBinaryComparators.scala

```diff
@@ -0,0 +1,12 @@
+package com.twitter.scalding.thrift.macros
+
+import com.twitter.scalding.serialization.{ OrderedSerialization, RequiredBinaryComparatorsConfig }
+import com.twitter.scalding.thrift.macros.impl.ScroogeInternalOrderedSerializationImpl
+import scala.language.experimental.{ macros => smacros }
+
+/**
+ * @author Mansur Ashraf.
+ */
+trait RequiredBinaryComparators extends RequiredBinaryComparatorsConfig {
```

Comment:
Can you add a comment to make it clear that this is here for people that use scrooge, but otherwise is identical to the one in scalding-core.